python/run.n

```diff
@@ -51,9 +51,16 @@ let ch:char = \n
 
 let lis = [1, 2, 3, 4]
 
-let list:list[func] = [main]
+// let list:list[func] = [main]
 
-var list = [1]
+// var list = [1]
+
+let record = {
+  ch
+  anyways: main; wo2w: true
+}
+print(record)
+//print ({ nested: [(record, 3)] } + 1)
```

Comment:
what is ch?

Reply:
Above, `let ch:char = \n` I believe

Reply:
oh ok, but why does it not have a key?

Reply:
`ch` on its own is implicitly a shorthand for `ch: ch`